### PR TITLE
[modular]ihopeits[ready] adds tuna and shortbread

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

shortbread and tuna, one is one of the greatest inventions to grace this wasteland known as sol 3
and the other is fish in a can
the latter, felines will most likely enjoy (and hopefully runtime, if someone who can actually code wants to)

## How This Contributes To The Skyrat Roleplay Experience

more food variety is good, isn't it?

## Changelog

:cl:
expansion: adds canned tuna and shortbread
/:cl:

### CODE REVIEW

**Comment by thestubborn** [comment]
image
the f
ood

File: modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm [hunk 1]
```diff
@@ -34,3 +34,28 @@
...
+/obj/item/food/cookie/shortbread
+	name = "shortbread"
+	desc = "A rectangular piece of cooked flour. Said to control the sun during Hogmanay."
+	icon = 'modular_skyrat/master_files/icons/obj/food/irnbru.dmi'
+	icon_state = "shortbread"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/consumable/sugar = 6)
+	tastes = list("sugar-y dough" = 1)
+	foodtypes = GRAIN | JUNKFOOD | SUGAR
+
+//HERE OUT OF LAZINESS/NOT SCOTTISH
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm]: Put it in an appropriate file

File: modular_skyrat/modules/customization/datums/components/crafting/recipes.dm [hunk 2]
```diff
@@ -31,3 +31,23 @@
 	)
 	result = /obj/item/food/sausage/battered
 	subcategory = CAT_MISCFOOD
+
+/datum/crafting_recipe/food/shortbread
+	time = 5
+	name = "Shortbread"
```
Comment (tf-4) [inline modular_skyrat/modules/customization/datums/components/crafting/recipes.dm]:
```suggestion
	name = "Shortbread"
	time = 5
```
Just for consistency.

File: modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm (hunk 1 above)
Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm]: ok babe

**Comment by YakumoChen** [comment]
Upstream first?

**Comment by thestubborn** [comment]
> Upstream first?

isnt upstream on feature freeze?
and sides, i dont think they'd like a two item pr

**Comment by Zonespace27** [comment]
The bottom of the shortbreads south-west corner looks a bit off and inflated to me, might just be me though

File: modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm (hunk 1 above)
Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm]: @tf-4 babe they're separate now...wanna kiss???

**Review by tf-4 — CHANGES_REQUESTED** [review]
Needs in-game screenshots, and tbh I'm not a huge fan of the sprites.

**Comment by thestubborn** [comment]
@tf-4 
image

File: modular_skyrat/modules/customization/datums/components/crafting/recipes.dm [hunk 3]
```diff
@@ -31,3 +31,23 @@
...
+	subcategory = CAT_PASTRY
+
+/datum/crafting_recipe/food/tuna
+	name = "Can of Tuna"
+	time = 40
+	reqs = list(/obj/item/stack/sheet/iron = 1,
+		/obj/item/food/fishmeat = 2
+	)
+	result = /obj/item/food/canned/beans
+	subcategory = CAT_MISCFOOD
```
Comment (thestubborn) [inline modular_skyrat/modules/customization/datums/components/crafting/recipes.dm]:
```suggestion
	subcategory = CAT_SEAFOOD
```

File: modular_skyrat/modules/customization/datums/components/crafting/recipes.dm [hunk 4]
```diff
@@ -31,3 +31,23 @@
...
+	result = /obj/item/food/cookie/shortbread
+	subcategory = CAT_PASTRY
+
+/datum/crafting_recipe/food/tuna
+	name = "Can of Tuna"
+	time = 40
+	reqs = list(/obj/item/stack/sheet/iron = 1,
+		/obj/item/food/fishmeat = 2
+	)
+	result = /obj/item/food/canned/beans
```
Comment (thestubborn) [inline modular_skyrat/modules/customization/datums/components/crafting/recipes.dm]:
```suggestion
	result = /obj/item/food/canned/tuna
```

File: modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm [hunk 5]
```diff
@@ -34,3 +34,11 @@
 	icon_state = "batteredsausage"
 
 
+/obj/item/food/cookie/shortbread
+	name = "shortbread"
+	desc = "A rectangular piece of cooked flour. Said to control the sun during Hogmanay."
+	icon = 'modular_skyrat/master_files/icons/obj/food/irnbru.dmi'
+	icon_state = "shortbread"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/consumable/sugar = 6)
+	tastes = list("sugar-y dough" = 1)
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm]:
```suggestion
	tastes = list("sugary dough" = 1)
```

**Pull request merged**